data-prepper-plugins/truncate-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/truncate/TruncateProcessorConfig.java

```diff
@@ -26,14 +24,18 @@ public static class Entry {
         @JsonProperty("length")
         private Integer length;
 
+        @JsonProperty("do_recursively")
```

Comment:
This seems a bit verbose, maybe "recursive" is clear enough?

Reply:
@oeyh , I like that suggestion.